docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Configure Teleport to create host users
+description: How to configure Teleport to automatically create transient host users.
+---
+
+Teleport's SSH service can be configured to automatically create local Unix users
+upon login.
+
+Host users created by Teleport are transient and will be deleted at the end of an SSH
+session.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+The following utilities should be available in the PATH:
```

Comment:
I think the utility names should be in code style so it's clearer that these are the names of the binaries you would execute on your system.

Reply:
Also, does the Teleport SSH Service require these utilities to be in PATH? I don't see examples of running these commands in this guide, so I'm not sure why the Prerequisites section requires them.